demo/tsconfig.json

```diff
@@ -18,6 +18,6 @@
       "@src/*": ["../src/*"]
     }
   },
-  "include": ["src"],
+  "include": ["src", "externals.d.ts"],
```

Comment:
why? where is this externals.d.ts in demo/

Reply:
I add an externals.d.ts file in demo/